code/modules/assembly/igniter.dm

```diff
@@ -22,7 +22,7 @@
 	if(!..())	return 0//Cooldown check
 	var/turf/location = get_turf(loc)
 	if(location)	location.hotspot_expose(1000,1000)
-	sparks.start()
+	if(sparks)	sparks.start()

```

Comment:
Why would sparks be null? If that shouldn't be happening then this is a bandaid and not an actual solution